docs: last storage plugin is default - minio (#1949)
medusajs/medusa: docs/content/add-plugins/minio.md

},
```

:::caution

> When you have multiple storage plugins configured, the one declared last in the `medusa-config.js` file will be used.

Reviewer: ```suggestion
If you have multiple storage plugins configured, the last plugin declared in the `medusa-config.js` file will be used.
```
